pkg/booleanpolicy/augmentedobjs/construct.go

```diff
@@ -150,7 +151,10 @@ func ConstructDeployment(deployment *storage.Deployment, images []*storage.Image
 	}
 
 	for i, image := range images {
-		augmentedImg, err := ConstructImage(image)
+		// Since we ensure that both images and containers have the same length, this will not lead to index out of
+		// bounds panics.
+		containerImageFullName := deployment.GetContainers()[i].GetImage().GetName().GetFullName()
```

Comment:
Daniel, could you help improve the documentation of this function? Adding a comment on `ln139` the function to indicate that it requires `images` to be ordered according to the container order in the `deployment` would make the requirement clearer.